src/libfsm/print/c.c

```diff
@@ -423,7 +421,9 @@ fsm_print_c(FILE *f, const struct fsm *fsm)
 		fprintf(f, "{\n");
 		fprintf(f, "\tint c;\n");
 		fprintf(f, "\n");
-		fprintf(f, "\tassert(fsm_getc != NULL);\n");
+		fprintf(f, "\tif (fsm_getc == NULL) {\n");
```

Comment:
Is fsm_getc being NULL here potentially a recoverable error?

Reply:
Undefined behavior it is! dc52d7c